pino.js

```diff
@@ -172,7 +172,7 @@ Pino.prototype.asJson = function asJson (obj, msg, num) {
     }
     for (var key in obj) {
       value = obj[key]
-      if (obj.hasOwnProperty(key) && value !== undefined) {
+      if (!obj.hasOwnProperty || obj.hasOwnProperty(key) && value !== undefined) {
```

Comment:
I'd use parentheses around that condition.
`(!obj.hasOwnProperty || obj.hasOwnProperty(key)) && value !== undefined`